src/util/Options.js

```diff
@@ -115,8 +116,8 @@ class Options extends null {
       messageSweepInterval: 0,
       invalidRequestWarningInterval: 0,
       partials: [],
-      restWsBridgeTimeout: 5_000,
-      restRequestTimeout: 15_000,
+      restWsBridgeTimeout: 5000,
+      restRequestTimeout: 15000,
```

Comment:
I don't think we want to revert those:
```suggestion
      restWsBridgeTimeout: 5_000,
      restRequestTimeout: 15_000,
```